### ISSUE TITLE
segmentation fault occured after last epoch

### ISSUE THREAD

**Issue by ziippy** [open-mmlab/mmrotate#484]
I'm running mmrotate on ubuntu 22.04

I train the ssdd_tiny dataset like below.
./tools/dist_train.sh ./configs/oriented_rcnn/oriented_rcnn_ssdd_tiny_copy.py 1

I want to check the operation quickly. so, I set the max epoch is 2

But, after 2 epoch -> saving checkpoint ok -> eval ok -> after then got a segmentation fault
what's the matter?

2022-08-29 11:33:40,126 - mmrotate - INFO - Saving checkpoint at 2 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 84/84, 16.7 task/s, elapsed: 5s, ETA:     0s

2022-08-29 11:33:50,910 - mmrotate - INFO - 
+-------+-----+------+--------+-------+
| class | gts | dets | recall | ap    |
+-------+-----+------+--------+-------+
| ship  | 176 | 615  | 0.176  | 0.046 |
+-------+-----+------+--------+-------+
| mAP   |     |      |        | 0.046 |
+-------+-----+------+--------+-------+
2022-08-29 11:33:50,942 - mmrotate - INFO - Exp name: oriented_rcnn_ssdd_tiny_copy.py
2022-08-29 11:33:50,943 - mmrotate - INFO - Epoch(val) [2][84]	mAP: 0.0461
./tools/dist_train.sh: line 20: 17182 Segmentation fault      (core dumped) PYTHONPATH="$(dirname $0)/..":$PYTHONPATH python -m torch.distributed.launch --nnodes=$NNODES --node_rank=$NODE_RANK --master_addr=$MASTER_ADDR --nproc_per_node=$GPUS --master_port=$PORT $(dirname "$0")/train.py $CONFIG --seed 0 --launcher pytorch ${@:3}
(mmrotate) ubuntu@ip-20-0-0-9:/mnt/disk2/mmrotate$ 

Why the segmentation occured?
How can I debug about that?

**Comment by ziippy** [comment]
This is another segment fault

2022-08-29 11:49:39,952 - mmrotate - INFO - Epoch [1][260/915]	lr: 1.697e-03, eta: 2:35:41, time: 0.267, data_time: 0.009, memory: 13398, loss_rpn_cls: 0.2763, loss_rpn_bbox: 0.3601, loss_cls: 0.3099, acc: 91.2988, loss_bbox: 0.1656, loss: 1.1119, grad_norm: 5.1319
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: -9) local_rank: 0 (pid: 17490) of binary: /home/ubuntu/.conda/envs/mmrotate/bin/python
./tools/dist_train.sh: line 20: 17487 Segmentation fault      (core dumped) PYTHONPATH="$(dirname $0)/..":$PYTHONPATH python -m torch.distributed.launch --nnodes=$NNODES --node_rank=$NODE_RANK --master_addr=$MASTER_ADDR --nproc_per_node=$GPUS --master_port=$PORT $(dirname "$0")/train.py $CONFIG --seed 0 --launcher pytorch ${@:3}

please advise about this issue.

**Comment by ziippy** [comment]
I'm using ﻿gpu_assign_thr=1750 for V100 16G
after then, the others running on CPU ---> But, my PC memory OOM too.

So, it occured the segmentation fault.
![image](https://user-images.githubusercontent.com/19854237/187200263-d916be2c-57ba-4f4f-8443-479e8d6fcf7f.png)
